docs/Project.toml

```
@@ -2,4 +2,4 @@
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 
 [compat]
-Documenter = "0.27"
\ No newline at end of file
+Documenter = "0.27, 1"
```

Since we don't use `strict = true` anymore (the default on Documenter 1, and not a kwarg anymore), we might also want to drop "support" for Documenter < 1:
```suggestion
Documenter = "1"
```